web/public/locales/es.json

```diff
@@ -78,6 +78,46 @@
     "fully-estimated": "Estimado"
   },
   "estimation-card": {
+    "link": "Más información sobre nuestros modelos de estimación",
+    "estimated_time_slicer_average": {
+      "title": "Los datos son estimados",
+      "pill": "Retraso",
+      "body": "Los datos de esta hora aún no se han transmitido. Los valores mostrados son estimaciones y se reemplazarán por mediciones cuando estén disponibles."
+    },
+    "estimated_mode_breakdown": {
+      "title": "Datos parcialmente estimados",
+      "pill": "Datos incompletos",
+      "body": "Los datos de generación de electricidad correspondientes a zona son incompletos.Los valores faltantes se han calculado mediante un modelo de estimación."
+    },
+    "estimated_reconstruct_breakdown": {
+      "title": "Los datos son siempre estimados",
+      "pill": "Datos imprecisos",
+      "body": "Los datos publicados correspondientes a  esta zona sólo proporcionan la generación total. Las fuentes renovables se estiman utilizando datos meteorológicos en tiempo real, mientras que las fuentes fósiles se deducen basándose en los datos históricos de cada fuente de producción."
+    },
+    "estimated_generic_method": {
+      "title": "Los datos son estimados",
+      "pill": "Datos imprecisos",
+      "body": "Los datos publicados para esta zona no están disponibles o son incompletos. Los datos que aparecen en el mapa se han estimado utilizando nuestro mejor esfuerzo, pero podrían diferir de los valores reales."
+    },
+    "estimated_construct_breakdown": {
+      "title": "Los datos son siempre estimados",
+      "pill": "Datos no disponibles en tiempo real",
+      "body": "Los datos para esta zona no están disponibles en tiempo real y sólo proporcionan la generación total. Tanto los valores horarios como el desglose por fuente de generación se estiman a partir de datos históricos."
+    },
+    "estimated_construct_zero_breakdown": {
+      "title": "Los datos son siempre estimados",
+      "pill": "Datos no disponibles en tiempo real",
+      "body": "Los datos para esta zona no están disponibles en tiempo real y sólo proporcionan la generación total. Tanto los valores horarios como el desglose por fuente de generación se estiman a partir de datos históricos."
+    },
+    "threshold_filtered": {
+      "title": "Datos no disponibles",
+      "pill": "Corte de datos",
+      "body": "El proveedor de datos para esta zona ha estado perdiendo datos durante un período prolongado. Estamos supervisando el problema y la zona se actualizará en cuanto los datos vuelvan a estar disponibles."
+    },
```

Comment:
We kind of stopped using this one as it was used when we were filtering prolonged estimates that we know are probably not accurate as it had the unintended consequence of marking the datapoints as estimated with out current logic.

So this one is not really needed anymore but I'm also not sure if we need to remove it since we might adopt another strategy for informing the users somehow.

Reply:
I have kept it for now and changed it to reflect your comments @wobniarin